python/cugraph/cugraph/tests/mg/test_mg_batch_betweenness_centrality.py

```diff
@@ -12,15 +12,18 @@
 # limitations under the License.
 
 import gc
+import sys
 
 import pytest
 import numpy as np
 
 from cugraph.dask.common.mg_utils import is_single_gpu
-from cugraph.tests.utils import RAPIDS_DATASET_ROOT_DIR_PATH
+from cugraph.testing.utils import RAPIDS_DATASET_ROOT_DIR_PATH
 
 # Get parameters from standard betwenness_centrality_test
-from cugraph.tests.test_betweenness_centrality import (
+# As tests directory is not a module, we need to add it to the path
+sys.path.insert(0, '../')
```

Comment:
A few things:
* this does not add the tests directory to the import path, it adds `..`, which will be interpreted as the parent directory of the CWD from where pytest is run.
* pytest automatically adds the directories containing the collected test modules to the import path, so this may be "accidentally" working because of that
* these changes are not being tested by CI since our test script skips everything in the `mg` directory, so have you manually tested these changes on a multi-GPU machine to ensure they work?
* needing to modify sys.path directly could be a symptom that we have a broken package hierarchy or some other code smell - is there a reason this directory is no longer a package (ie. had its `__init__.py` file removed)?

Reply:
@Ethyling based on [our conversation](https://github.com/rapidsai/cugraph/pull/2230#issuecomment-1123923864), would you mind removing the `sys.path.insert()` calls and adding a `FIXME` stating that those tests need to be refactored to not import other tests and should instead put shared test code in a util module? Same for the other comment below.